### PR TITLE
[*] CORE: Fix undefined PHPDoc types of method and local variables + remove static tag

### PR DESCRIPTION
**Pull request by gskema**
- static tag is redundant, static methods and variables are detected without this tag;
- PHPDocs for local scope variables is very useful for programmers and for completing PHPDocs of functions;

### CODE REVIEW

File: classes/Product.php [hunk 1]
```diff
@@ -2711,25 +2718,31 @@ public static function getPriceStatic($id_product, $usetax = true, $id_product_a
...
-	* @return float Product price
-	**/
+	 * Price calculation / Get product price
+	 *
+	 * @param int    $id_shop Shop id
+	 * @param int    $id_product Product id
+	 * @param int    $id_product_attribute Product attribute id
+	 * @param int    $id_country Country id
+	 * @param int    $id_state State id
+	 * @paran string $zipcode
```
Comment (jeromenadaud) [inline classes/Product.php]: Hi, must be ```@param```, not ```@paran```

File: classes/helper/HelperView.php [hunk 2]
```diff
@@ -31,7 +31,7 @@ class HelperViewCore extends Helper
 	public $table;
 	public $token;
 
-	/** @var if not null, a title will be added on that list */
+	/** @var string|null Ff not null, a title will be added on that list */
```
Comment (jeromenadaud) [inline classes/helper/HelperView.php]: Hi, If not null, not Ff not null ;-)

**Comment by gskema** [comment]
Hello,

is everything else ok? I can't fix the typos atm, I'm on a different computer.

**Comment by jeromenadaud** [comment]
Hi, 
Yes everything else is OK.

Best regards.

**Comment by gskema** [comment]
Hello,

I updated the pull request.
- fixed typos
- moved some inline PHPDoc under if statments (PHPDoc respects scopes)

**Comment by jeromenadaud** [comment]
Hi,

Thank you for your great contribution. ;-)

Kindly regards.

**Pull request merged**